tools/test_configs/EthernetInterface.json

```diff
@@ -10,18 +10,6 @@
         "connect-statement" : {
             "help" : "Must use 'net' variable name",
             "value" : "((EthernetInterface *)net)->connect()"
-        },
```

Comment:
These changes will cause mbed OS tests to fail. I believe this should continue to be the default test configuration file. If echo server changes are necessary, then that person should use a custom test configuration file and specify it with the `--test-config` option.

Reply:
No...
Check the updates in test c files. You can keep these setting in your files, there will be no change for you.
I user doesn't want to set manually the settings and use greentea feature, it is now possible

Reply:
Yes. By default, LWIP targets use this file. The server is not set to ublox echo server in this file, nor in the cpp files.

Reply:
Change your test setup...
Default configuration should be the basic configuration without any specific setting.
If you have a specific test environment, use a specific json configuration file.

Reply:
By and large, the Ublox server will be accessible on a network. It is a special case that is is not accessible. We want to provide a sensible default, and it is, unless you have a special case.